crates/globwatch/readme.md

```diff
@@ -0,0 +1,27 @@
+# Globwatch
+
+> Watch a set of globs efficiently
+
+This library provides an async interface over notify and glob-match to
+efficiently watch a list of globs. Where possible it attempts to minimize the
```

Comment:
```suggestion
watch a list of globs. Where possible it attempts to minimize the
```

nit: redundant. I don't think anyone would ever claim the opposite :). And the explanation of how it's efficient is already way better than the claim!